test/Npgsql.Tests/BugTests.cs

```diff
@@ -1238,5 +1238,43 @@ public void NullReferenceExceptionInBeginTextExport()
                 conn.ExecuteNonQuery("DROP FUNCTION IF EXISTS f_test()");
             }
         }
+
+        public enum TestEnum
+        {
+            One,
+            Two
+        }
+
+        class SomeComposite
+        {
+            /// <summary>
+            /// An enum without proper handler
+            /// </summary>
+            public TestEnum Test { get; set; }
+            public int X { get; set; }
+            public string SomeText { get; set; } = "";
+        }
+
+        [Test]
+        public void CompositePostgresType()
```

Comment:
How is this test related to this PR?

Reply:
This is the test replicating the initial issue #2621 this PR comes from.
I propose to keep it.